usr/share/rear/conf/default.conf

```diff
@@ -1478,6 +1478,18 @@ LANG_RECOVER=C
 # low-quality master encryption key. For details, see the cryptsetup(8) manual page.
 LUKS_CRYPTSETUP_OPTIONS="--iter-time 2000 --use-random"
 
+##
+# BACKUP=CDM (Rubrik CDM; Cloud Data Managemnt)
+##
+#
+COPY_AS_IS_CDM=( /etc/rubrik /usr/bin/rubrik /var/log/rubrik /etc/pki /usr/lib64 )
+COPY_AS_IS_EXCLUDE_CDM=( /var/log/rubrik/* )
+PROGS_CDM=( /usr/bin/rubrik/backup_agent_main /usr/bin/rubrik/bootstrap_agent_main openssl uuidgen )
+
+#COPY_AS_IS_CDM=( /usr/lib/rpm /usr/lib/modules /etc/pki /usr/lib64 )
+#COPY_AS_IS_EXCLUDE_CDM=( )
+#PROGS_CDM=(  rpm convertdb1 gendiff rpm2cpio rpmdb rpmgraph rpmkeys rpmqpack rpmquery rpmsign rpmspec rpmverify rcrpmconfigcheck openssl )
```

Comment:
@DamaniN 
please add an explanatory comment about what the reasoning behind is 
why there is this other block with commented values in default.conf,
cf. https://github.com/rear/rear/wiki/Coding-Style

Reply:
Removed commented block. This was used for development. I also added comments about what CDM does.